resolver/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "commonware-resolver"
+edition = "2021"
+publish = true
+version = "0.0.2"
+license = "MIT OR Apache-2.0"
+description = "Resolve data by unique key."
```

Comment:
nit: `Resolve data from untrusted peers by key.`

Reply:
This is the comment for resolver in general, which may not use peers if I understand correctly. So that's why I left it vague. I could even see there being a resolver for something like local storage where it doesn't use an external party at all